ios/example/tests/rotateAnnotation.js

```diff
@@ -0,0 +1,42 @@
+exports.title = 'rotateAnnotation';
+exports.run = function(UI, Map) {
+var win = UI.createWindow(exports.title);
+var map = Map.createView({
+    region: {
+        latitude: 45,
+        longitude: -76,
+        latitudeDelta: 0.08,
+        longitudeDelta: 0.08
+    },
+    bottom: 40
+});
+win.add(map);
+
+var button = Ti.UI.createButton({
+    title: "update",
+    bottom: 5
+});
+win.add(button);
+button.addEventListener('click', function() {
+    updateAnnotations();
+});
+win.open();
+
+var annotation = null;
+createAnnotations();
+
+function createAnnotations() {
+    annotation = Map.createAnnotation({
+            latitude: 45,
+            longitude: -76,
+            title: "annotation "
+    });
+    map.addAnnotation(annotation);
+}
+
+function updateAnnotations() {
+    console.log("updateAnnotations");
+    var destLatLong = [45.05,-76.05];
+    map.animateAnnotation([annotation, destLatLong]);
```

Comment:
There is no API named 'animateAnnotation'. It is 'animate' on Annotation. Please verify test cases at your end.

Reply:
Add a similar test case for rotation as well.

Reply:
I am added test cases in annotation.js file so I am removing this rotateAnnotation.js file as its no longer needed.